packages/manager/modules/pci/src/projects/project/data-processing/notebooks/add-notebook/translations/Messages_fr_FR.json

```
@@ -0,0 +1,44 @@
+{
+  "data_processing_add_notebook_title": "Créer un notebook",
+  "data_processing_add_notebook_description": "Démarrez vos notebooks Jupyter dans le cloud et lancez des jobs Apache Spark à la demande depuis ceux-ci.",
+  "data_processing_add_notebook_description_warning_ai": "Vous cherchez plutôt à explorer vos données via un notebook permettant d'exploiter les plus grands frameworks d'intelligence artificielle (Tensorflow, PyTorch, Hugging Face ou encore Scikit-learn).",
+  "data_processing_add_notebook_description_warning_ai_link": "Essayez les AI Notebooks",
+  "data_processing_add_notebook_stepper_region_title": "Sélectionner une région",
+  "data_processing_add_notebook_stepper_notebook_type_title": "Sélectionner un framework",
+  "data_processing_add_notebook_stepper_sizing_title": "Dimensionnement du job",
+  "data_processing_add_notebook_stepper_privacy_title": "Paramètres de confidentialité",
+  "data_processing_add_notebook_stepper_name_title": "Nommer le notebook",
+  "data_processing_add_notebook_stepper_submit_title": "Soumettre votre job",
+  "data_processing_add_notebook_stepper_api_title": "Informations de votre notebook",
+  "data_processing_add_notebook_command_description": "La commande suivante vous permet de créer votre notebook grâce à notre API.",
+  "data_processing_add_notebook_command_documentation_link": "Cliquez-ici pour accéder à la documentation.",
+  "data_processing_add_notebook_stepper_submit_error_title": "Une erreur est survenue",
+  "data_processing_add_notebook_button_label": "Créer le notebook",
+  "data_processing_add_notebook_spinner_label": "lancement en cours…",
+  "data_processing_add_notebook_error_label": "Une erreur est survenue, nous ne pouvons pas démarrer votre job pour le moment.",
+  "data_processing_add_notebook_spec_heading": "Votre notebook",
+  "data_processing_add_notebook_spec_name": "Nom : ",
+  "data_processing_add_notebook_spec_engine": "Type :",
```

To be consistent with others:

```suggestion
  "data_processing_add_notebook_spec_engine": "Type : ",
```

---

Fixed :)